database/sqlite/sqlite_aclk_chart.c

```diff
@@ -342,6 +347,7 @@ void aclk_send_chart_event(struct aclk_database_worker_config *wc, struct aclk_d
 #else
     UNUSED(wc);
     UNUSED(cmd);
+    info("ACLK_NG not defined");
```

Comment:
ACLK-NG not available in this agent or sth. more friendly?

Reply:
I will either update the message or remove it .. It shouldn't even reach that point if aclk is not available